qiskit_machine_learning/algorithms/regressors/qsvr.py

```diff
@@ -45,9 +46,20 @@ def __init__(self, *args, quantum_kernel: Optional[QuantumKernel] = None, **kwar
             **kwargs: Arbitrary keyword arguments to pass to SVR constructor.
         """
 
+        if (len(args)) != 0:
+            msg = (
+                f"Positional arguments ({args}) are deprecated as of version 0.3.0 and "
+                f"will be removed no sooner than 3 months after the release. Instead use "
+                f"keyword arguments. Positional arguments are ignored and not passed to the superclass."
```

Comment:
Similar comment for ` Positional arguments are ignored and not passed to the superclass.`